### PR TITLE
test: unit tests for the cli

### PR DESCRIPTION
**Pull request by antialias**

### CODE REVIEW

File: lib/main.test.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                                            schema testing.
+        -t, --theme <theme name>            Specify theme used by \`export\` and
+                                            \`serve\` or specify a path starting with .
+                                            (use . for current directory or
+                                            ../some/other/dir) (default:
+                                            \\"jsonresume-theme-even\\")
+        -f, --format <file type extension>  Used by \`export\`.
+        -r, --resume <resume filename>      path to the resume in json format
+                                            (default:
+                                            \\"/Users/thomas.hallock/projects/resume-cli/resume.json\\")
```
Comment (rbardini) [inline lib/main.test.js:30]: Won't this line fail in other environments? We probably need to update our `test` job to run on forks as well.

Comment (antialias) [reply lib/main.test.js:30]: oops, I didn't catch that. stand by for an update...

Comment (antialias) [reply lib/main.test.js]: now using relative path in --help output

**Review by thomasdavis — APPROVED** [review]
LGTM

**Comment by rbardini** [comment]
Test step is failing on `master` since 245ea94 (probably introduced by 6c9b0d1, but it failed earlier). Does this PR fix the issue?

Only our PR workflow is running on pull requests from forks, that's probably why we didn't catch it before merging.

**Comment by antialias** [comment]
> Test step is failing on `master` since [245ea94](https://github.com/jsonresume/resume-cli/commit/245ea9495b9070ca616a9089b5efe56cf09067fd) (probably introduced by [6c9b0d1](https://github.com/jsonresume/resume-cli/commit/6c9b0d1e9b76463056a5ed901d0f8e8b086cf016), but it failed earlier). Does this PR fix the issue?
> 
> Only our PR workflow is running on pull requests from forks, that's probably why we didn't catch it before merging.

PR that fixes regression https://github.com/jsonresume/resume-cli/pull/463

**Pull request merged**